maas-schemas-ts/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "maas-schemas-ts",
-  "version": "14.10.0",
+  "version": "14.10.1",
```

Comment:
Technically this might be more of a minor version bump rather than a patch bump? Up to you @thanhtr